File: DatabaseContractor/FilterModel.cs

using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;

namespace DatabaseContractor
{
    [ExcludeFromCodeCoverage]
    public class FilterModel
    {
        public List<string> DisplayType { get; set; }
        public Limits DisplaySize { get; set; }
        public Limits Weight { get; set; }
        public bool TouchScreen { get; set; }

        
    }
    [ExcludeFromCodeCoverage]
        public class Limits
        {
            
            public string Max { get; set; }
            public string Min { get; set; }
        }
    

       
    
}

File: app2/src/app/FilterModel.ts
import { EmailValidator } from '@angular/forms';
import { ProductDetails } from './ProductDetails';

  export class FilterModel
  {
    DisplayType:string[];
    DisplaySize:Limits;
    Weight:Limits;
    TouchScreen:boolean;
  }

 export  class Limits
      {
        max:Int32Array;
        min:Int32Array;
         
      }


  export class UserForSalesDetails{
    CustomerName:string;
    EmailId:EmailValidator
    Description:ProductDetails[]

}
     